src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -132,12 +132,12 @@ export const downloadOptions = [
         externalLink: false
     },
     {
-        label: 'Metadata Download',
+        label: 'Dataset Metadata',
         type: 'metadata_download',
         url: '/download_center/metadata_download',
```

Comment:
Wondering if we want to also change the url to `/dataset_metadata`

Reply:
updated!